### ISSUE TITLE
Add a clear and fitting title for the feature request

### ISSUE THREAD

**Issue by mau316** [CrocoBlock/suggestions#6167]
If there is an option in the forms to add a checkbox or a select from **query builder**
That will be great

**Comment by edgBllw** [comment]
Yes, actually it's already possible. Lets say i created a query with the query builder that lists all users of the site. If you want do populate these within a checkbox or select field in a jetformbuilder form. What you need to do is following:

1.) Create form via JetFormBuilder
2.) Create checkbox or select field
3.) For "Fill Options From" choose "Generate Dynamically"
4.) For "Generator Fuction" choose "Get values list from JetEngine Query"
5.) For "Field Name" you need to input following format: Query ID|Option Value|Option Name

So for query ID hop on to the query builder and hover over the desired query you want to show inside the form. You should see at the very left bottom of your page a link which ends with "&id=" . The id number which is shown after this string is the id you need.

For example, if my users query has the id of 9, and in my form i want the users id as value for the option and the display name as option name, my input for "Field Name" would look like this: 9|ID|display_name


You just need to know the wordpress object property keys for users or posts or what you want...

A few for users are:
ID
first_name
last_name
user_email
user_login
display_name
...

A few for posts are:
ID
post_title
post_name
post_date
post_author
post_status
...

I hope that helps..
